test-data/unit/check-python38.test

```diff
@@ -181,3 +181,122 @@ def f(p1: bytes, p2: float, /, p_or_kw: int) -> None:
 def f(p1: bytes, p2: float, /) -> None:
     reveal_type(p1)  # E: Revealed type is 'builtins.bytes'
     reveal_type(p2)  # E: Revealed type is 'builtins.float'
+
+[case testWalrus]
+from typing import NamedTuple
+
+if (a := 2):
```

Comment:
I think that the parentheses are optional here. Maybe leave out all optional parentheses?